analytics.js

```diff
@@ -68,6 +80,25 @@ function pageview(url) {
 }
 
 /**
+ * Set a custom dimension for future page views and events
+ * @param name
+ * @param cookieId
+ */
+function dimensionByCookie(name, cookieId) {
```

Comment:
Can we generalize this to a `dimension` method, so we could set other dimensions in the future?

```js
dimension('dimensionName', getCookie('cookieName'));
```